channeldb/graph.go

```diff
@@ -439,12 +446,29 @@ func (c *ChannelGraph) SetSourceNode(node *LightningNode) error {
 	})
 }
 
+// BatchedAddLightningNode adds a vertex/node to the graph database. If the node
+// is not in the database from before, this will add a new, unconnected one to
+// the graph. If it is present from before, this will update that node's
+// information. Note that this method is expected to only be called to update an
+// already present node from a node announcement, or to insert a node found in a
+// channel update.
+func (c *ChannelGraph) BatchedAddLightningNode(node *LightningNode) error {
+	return c.nodeScheduler.Execute(&batch.Request{
+		Update: func(tx *bbolt.Tx) error {
+			return addLightningNode(tx, node)
```

Comment:
Should some errors be suppressed to let the batch go through?

Reply:
`addLightningNode` only produces critical errors, e.g. serialization, bucket structure, etc, so we don't need to catch anything afaict